apps/src/netsim/netsim.js

```diff
@@ -304,7 +322,9 @@ NetSim.prototype.changeRemoteDnsMode = function (newDnsMode) {
  * @param {boolean} isDnsNode
  */
 NetSim.prototype.setIsDnsNode = function (isDnsNode) {
-  this.tabs_.setIsDnsNode(isDnsNode);
```

Comment:
Great question to which I don't yet know the answer.  There's something nice about having explicit calls so you know who's talking to who, but there's also something nice about decoupling component dependencies.  I suspect the microservices world prefers the latter... going to a meetup tonight which might influence thinking :)